classes/startup.rb

```diff
@@ -1,8 +1,10 @@
-require_relative '../actions/game_actions'
+require './actions/game_actions'
```

Comment:
I don't know what  was wrong with this, but it was not working and since it was my own mistake, I just changed it 😁

Reply:
Oh sorry about that,I was having merge errors so I think I changed that by mistake 😊.

Thanks for correcting it